apps/client/jest.config.ts

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable */
```

Comment:
Find out what `jest` do
No need to change anything here

Reply:
Added to "things to learn list"